README.rst

```diff
@@ -60,7 +60,7 @@ How to install
 How to use
 ----------
 
-The main goal of ``GenPackageDoc`` is to document Python sources that are stored within a repository.
+``GenPackageDoc`` provides a toolchain to generate documentation out of Python sources that are stored within a repository.
```

Comment:
Hi Holger,
what about the RST and Latex files wich are used to generate the documentation?
Thank you,
  Thomas

Reply:
They are mentioned explicitely at the top of the README.